src/components/Assesment/Assesment.jsx

```diff
@@ -336,6 +336,19 @@ export const ProfileHeader = ({
   const username = profileName || getLocalData("profileName");
   const navigate = useNavigate();
   const [openMessageDialog, setOpenMessageDialog] = useState("");
+
+  const handleProfileBack = () => {
+    try {
+      if(localStorage.getItem("token")){
+        let length = 100; 
+        window.parent.postMessage({ type: 'stringLengths', length }, '*');
```

Comment:
stringLengths => "content-string-length" but you can not pass the hardcoded string length - there should be a another postmessage - like - restore-content-length OR restore-iframe-content OR content-refresh